### ISSUE TITLE
Any build notifications in Slack does not work, but the "Try It" function does.

### ISSUE THREAD

**Issue by sergey-prosvirnin** [alexkvak/teamcity-slack#100]
I have **TeamCity Professional 2019.2.1 (build 71758)** in which I used to have successful notifications when building successfully in Slack. But now they have stopped working. At the same time **Try It** function successfully works and sends test notifications to Slack.

Plugin version: 1.5.1
Java version: 13.0.2
OS: Ubuntu 16.04

![image](https://user-images.githubusercontent.com/24830345/75253298-4aaeee80-57ef-11ea-868a-898203dc30b7.png)
![image](https://user-images.githubusercontent.com/24830345/75253459-9f526980-57ef-11ea-8dcf-63e0e26d39c5.png)
![image](https://user-images.githubusercontent.com/24830345/75253527-bd1fce80-57ef-11ea-81b0-915089697950.png)

**Comment by alexkvak** [comment]
Could you check Administration -> Diagnostics -> Server Logs -> teamcity-server.log for any related to Slack errors?

**Comment by alexkvak** [comment]
Is notifications broken since update to 1.5.1?

**Comment by sergey-prosvirnin** [comment]
> Is notifications broken since update to 1.5.1?

No, it's still 1.5.0. I was trying to find a solution to the problem in the update.

**Comment by sergey-prosvirnin** [comment]
[teamcity-server.log](https://github.com/alexkvak/teamcity-slack/files/4250392/teamcity-server.log)


> Could you check Administration -> Diagnostics -> Server Logs -> teamcity-server.log for any related to Slack errors?

I looked quickly and found no Slack errors.

[teamcity-server.log](https://github.com/alexkvak/teamcity-slack/files/4250394/teamcity-server.log)

**Comment by alexkvak** [comment]
Could you attach screenshot with Slack build configuration? (Press Edit near Try it)

**Comment by sergey-prosvirnin** [comment]
> Could you attach screenshot with Slack build configuration? (Press Edit near Try it)

![image](https://user-images.githubusercontent.com/24830345/75257398-f3604c80-57f5-11ea-91d9-0185e804b13d.png)

**Comment by alexkvak** [comment]
Please look at #21 and #25

You should specify branch name according to the names you see in TeamCity build info.

I guess in your case correct name for Branch field is `development`

**Comment by sergey-prosvirnin** [comment]
![image](https://user-images.githubusercontent.com/24830345/75260626-f1e55300-57fa-11ea-8ac6-b144805d383d.png)

**Comment by alexkvak** [comment]
What branch name do you see in build history list?

**Comment by sergey-prosvirnin** [comment]
I guess I didn't quite get it right where it was.

![image](https://user-images.githubusercontent.com/24830345/75261555-4b01b680-57fc-11ea-97da-0697fd79880c.png)

**Comment by alexkvak** [comment]
As you can see, TeamCity does not parse branch name.

**Comment by sergey-prosvirnin** [comment]
I see the names of the repositories in the build log.

![image](https://user-images.githubusercontent.com/24830345/75262762-07a84780-57fe-11ea-9ca5-a9dfa13f2b99.png)
![image](https://user-images.githubusercontent.com/24830345/75262963-5950d200-57fe-11ea-9b9d-703eae7fd3fc.png)

**Comment by alexkvak** [comment]
You should edit VCS root and specify correct branch specification
﻿
![изображение](https://user-images.githubusercontent.com/1052418/75264785-fb71b980-5800-11ea-8255-a783ec56772c.png)

**Comment by sergey-prosvirnin** [comment]
Unfortunately, these actions have not been successful.

![image](https://user-images.githubusercontent.com/24830345/75268931-2f50dd00-5809-11ea-9962-a47cce6ea317.png)

**Comment by alexkvak** [comment]
![изображение](https://user-images.githubusercontent.com/1052418/75270267-527c8c00-580b-11ea-8d2e-315f7772ff4b.png)

as a result you can now find the branch name on build page

![изображение](https://user-images.githubusercontent.com/1052418/75270407-8f488300-580b-11ea-969f-ef7617edccbd.png)
